src/sinks/aws_cloudwatch_logs.rs

```diff
@@ -279,11 +431,16 @@ impl fmt::Display for CloudwatchError {
         match self {
             CloudwatchError::Put(e) => write!(f, "CloudwatchError::Put: {}", e),
             CloudwatchError::Describe(e) => write!(f, "CloudwatchError::Describe: {}", e),
+            CloudwatchError::CreateStream(e) => write!(f, "CloudwatchError::CreateStream: {}", e),
             CloudwatchError::NoStreamsFound => write!(f, "CloudwatchError: No Streams Found"),
             CloudwatchError::ServiceDropped => write!(
                 f,
                 "CloudwatchError: The service was dropped while there was a request in flight."
             ),
+            CloudwatchError::MakeService => write!(
+                f,
+                "CloudwatchError: The inner service was unable to be created."
```

Comment:
Is this a panic kind of situation? From a user's perspective, if I got this error I wouldn't know what to make of it.

Reply:
Yes, I believe this is when we cant spawn the request. The specific error type is to inform us where this error occurred and its a decently fatal type of error.

Reply:
Ok, I just want to make sure anything we spit out to the user makes sense to them and is ideally actionable.